repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -5,16 +5,18 @@
 from leapp.libraries.actor import constants
 from leapp.libraries.common import dnfplugin, mounting, overlaygen, rhsm, utils
 from leapp.libraries.common.config import get_product_type
-from leapp.libraries.stdlib import CalledProcessError, api, run, config
-from leapp.models import (RequiredTargetUserspacePackages, SourceRHSMInfo, StorageInfo, TargetRepositories,
-                          TargetUserSpaceInfo, UsedTargetRepositories, UsedTargetRepository, XFSPresence)
+from leapp.libraries.stdlib import CalledProcessError, api, config, run
+from leapp.models import (RequiredTargetUserspacePackages, RHSMInfo,
+                          StorageInfo, TargetRepositories, TargetRHSMInfo,
```

Comment:
regarding your latest changes, there is not any targetrhsminfo anymore.

Reply:
Yeah, that was a leftover. Thanks for pointing it out. It's removed now.